src/py_semantic_taxonomy/adapters/routers/router.py

```diff
@@ -12,19 +12,24 @@
 from py_semantic_taxonomy.dependencies import get_graph_service, get_search_service
 from py_semantic_taxonomy.domain import entities as de
 
-router = APIRouter()
+router = APIRouter(prefix="/v1")
 
 
 class Paths(StrEnum):
-    concept = "/concept/"
-    concept_scheme = "/concept_scheme/"
-    catchall = "/{_:path}"
-    relationship = "/relationships/"
-    correspondence = "/correspondence/"
-    association = "/association/"
-    made_of = "/made_of/"
-    search = "/concept/search/"
-    suggest = "/concept/suggest/"
+    concepts = "/concepts"
```

Comment:
After looking through the PR, given that we are all (?) humans, it's going to be super-easy to make mistakes if there is only one letter different. I would prefer `Paths.concept_list` instead of `Paths.concepts` - what do you think?

Reply:
Sounds good, but we might want to revisit whether we need the enum, given that there's less to share among paths now?